Documentation/gettingstarted/hubble.rst

```diff
@@ -31,8 +31,8 @@ Deploy Cilium and Hubble
 
 .. include:: k8s-install-download-release.rst
 
-Pre-load images into the kind cluster so each node does not have to pull
-them:
+**(optional, but recommended)** Pre-load Cilium images into the kind cluster so
```

Comment:
i slightly prefer not bothering with `(optional, but recommended)`. if it's better to do it, just list the command.

Reply:
I thought about it as well but it's out of scope of this PR. Will do in another PR that also remove duplicated instructions.

Reply:
Included in #12210.